blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -37,30 +37,14 @@ Unreleased Changes
 Breaking Changes
 ================
 
+None
```

Comment:
This is new to me, but I followed the instruction from here: https://github.com/crate/crate/pull/8094/files#diff-8918b36d12905ff85de79bcbdcf052aaR29